images/iqsharp-base/Dockerfile

```diff
@@ -124,7 +129,8 @@ RUN chown -R ${USER}:${USER} /home/${USER}/.azure
 USER ${USER}
 # Make sure that .NET Core is on the notebook users' path.
 ENV PATH=$PATH:${HOME}/dotnet:${HOME}/.dotnet/tools \
-    DOTNET_ROOT=${HOME}/dotnet
+    DOTNET_ROOT=${HOME}/dotnet \
+    PYTHONPATH=$PYTHONPATH:/usr/lib/python3/dist-packages/
```

Comment:
If possible, I'd recommend against doing this; the FROM line for this image bases on the `python:slim-buster` image, which maintains a copy of Python separate from the one used by the OS itself. Adding dist-packages could cause conflicts or other unexpected behavior by making all OS-level packages available to the separate install of Python added by the base image on the FROM line.

Reply:
I'm not sure that is a real risk here; it doesn't look like there's any overlap between `dist-packages` and `site-packages`.
An alternative solution would be to add a `.pth` file; e.g. `dist-packages.pth` to `/usr/local/lib/python3.7/site-packages` containing the path to `/gi`. Would that work for you?